Korean Translation of challenge "Pick"

Dogdriip opened the pull request:
Hello. I just translated the challenge "Pick" to Korean. 

Please note that this is my first contribution. If this is correct, I will continue to translate other challenges.

jiangshanmeta commented:
@Dogdriip Thanks for you contribution.

In order to add a new locale ,more mork needs to be done. You can see [file locales.ts](https://github.com/jiangshanmeta/type-challenges/blob/master/scripts/locales.ts#L5)

```typescript
export const messages = {
  en: require('./locales/en.json'),
  'zh-CN': require('./locales/zh-CN.json'),
  ja: require('./locales/ja.json'),
}
```

You need to update this file and add a *ko.json* file

Dogdriip commented:
> @Dogdriip Thanks for you contribution.
> 
> In order to add a new locale ,more mork needs to be done. You can see [file locales.ts](https://github.com/jiangshanmeta/type-challenges/blob/master/scripts/locales.ts#L5)
> 
> ```ts
> export const messages = {
>   en: require('./locales/en.json'),
>   'zh-CN': require('./locales/zh-CN.json'),
>   ja: require('./locales/ja.json'),
> }
> ```
> 
> You need to update this file and add a _ko.json_ file

Thanks for pointing it out. I just added _ko.json_ file and modified _locales.ts_ correctly.

Thanks in advance.
